### DISCUSSION TITLE
Usage with zod and sveltekit-superforms v2

### DISCUSSION THREAD

**Discussion by Sejmou** [huntabyte/shadcn-svelte#949] [Help]
Hey,

first of all, thank you for this awesome component library! It really helps me out a ton.

I also really enjoy the documentation. However, I struggle to follow the [guide for the Form component](https://www.shadcn-svelte.com/docs/components/form) (featuring `sveltekit-superforms`, `zod` and `formsnap`). I get type errors. For example, `superForm(data.form)` in my `+page.svelte` file doesn't contain the `formData` property.

Is this because the documentation is for v1 of `sveltekit-superforms`? Can I still somehow use v2 or will I have to downgrade? I would prefer using v2 as the `sveltekit-superforms` seem to use that (although they also aren't particularly transparent with that).

Thank you for taking some time to help me out :)

Greetings,
Samo

**Comment by ghost** [comment]
I'm getting the same error.

**Comment by nickolay** [comment]
FWIW, the current instructions seem to work for me with:

		"svelte": "^5.0.0-next.1",
		"formsnap": "^1.0.1",
		"sveltekit-superforms": "^2.19.1",

`superForm(data.form)` is [supposed to have](https://superforms.rocks/api#superform-return-type) a `form`, not `formData` property, though.
